### PR TITLE
The cyborg chameleon projector can now transform into every (crew) cyborg type

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
The cyborg chameleon projector can now transform into every (crew) cyborg type, rather than just the landmate cyborg
All things you would expect to change (Desc and icon) change upon use.
While already cloaking, it takes no time to change a disguise again
Instead of using the cloaker again to change back to normal, you hit the Cancel button
(The examine text for the saboborg has also been renamed to "saboteur module" but that's minor and it was only like that in the first place because you could only turn into an engineering borg)
## Why It's Good For The Game
The current iteration of this item sucks, like wholesale just not useful
You're locked into one module sprite, so you're 100% getting metachecked via a shove or a disabler beam
There's still signs that someone is a saboborg, such as "why is that mediborg holding an esword", but random landmates aren't going to get ioned.
Aside from that, it's just a fun tool. Using this for infiltration is still not gonna be easy, but it will be much more possible.

## Images of changes

https://user-images.githubusercontent.com/96800819/210465988-6614a8b3-b03f-40d8-b766-c831d9cdb59a.mp4

## Testing
Compiled, ran, am real borg you can trust

## Changelog
:cl:
tweak: The cyborg chameleon projector now can transfer into any crew obtainable borg
/:cl:

### CODE REVIEW

**Comment by SpringSkipper** [comment]
Awesome! They can still get outed by the AI, but this is a huge boon especially for the rare stealthops nukies.

File: code/game/objects/items/devices/chameleonproj.dm [hunk 1]
```diff
@@ -207,15 +207,12 @@
 
 /obj/item/borg_chameleon/proc/toggle(mob/living/silicon/robot/syndicate/saboteur/user)
 	if(active)
-		playsound(src, 'sound/effects/pop.ogg', 100, 1, -6)
-		to_chat(user, "<span class='notice'>You deactivate [src].</span>")
-		deactivate(user)
+		to_chat(user, "<span class='notice'>You reconfigure [src].</span>")
+		activate(user)
 	else
```
Comment (Sirryan2002) [inline code/game/objects/items/devices/chameleonproj.dm:212]: could you add an early return here?

File: code/game/objects/items/devices/chameleonproj.dm [hunk 2]
```diff
@@ -230,21 +227,58 @@
...
-	user.base_icon = disguise
-	user.icon_state = disguise
+	user.icon = disguise.icon
+	user.icon_state = disguise.icon_state
 	user.cham_proj = src
+	user.module.name = examine_text
 	user.bubble_icon = "robot"
 	active = TRUE
 	user.update_icons()
+	playsound(src, 'sound/effects/bamf.ogg', 100, 1, -6)
```
Comment (Sirryan2002) [inline code/game/objects/items/devices/chameleonproj.dm:271]:
```suggestion
	playsound(src, 'sound/effects/bamf.ogg', 100, TRUE, -6)
```

File: code/game/objects/items/devices/chameleonproj.dm (hunk 1 above)
Comment (GDNgit) [reply code/game/objects/items/devices/chameleonproj.dm:212]: applied

File: code/game/objects/items/devices/chameleonproj.dm (hunk 2 above)
Comment (GDNgit) [reply code/game/objects/items/devices/chameleonproj.dm]: applied

File: code/game/objects/items/devices/chameleonproj.dm (hunk 1 above)
Comment (farie82) [reply code/game/objects/items/devices/chameleonproj.dm:212]: I think you applied it to another proc. This one still could use an early return

Comment (GDNgit) [reply code/game/objects/items/devices/chameleonproj.dm:212]: I'll check when I get home (eta 6 hours)

**Review by S34NW — COMMENTED** [review]
Can't see any other issues apart from the missing early return mentioned above

File: code/game/objects/items/devices/chameleonproj.dm (hunk 1 above)
Comment (GDNgit) [reply code/game/objects/items/devices/chameleonproj.dm]: Yep I missed it, applied

**Review by S34NW — APPROVED** [review]

File: code/game/objects/items/devices/chameleonproj.dm [hunk 3]
```diff
@@ -209,15 +209,15 @@
 	if(active)
 		to_chat(user, "<span class='notice'>You reconfigure [src].</span>")
 		activate(user)
+		return
+	to_chat(user, "<span class='notice'>You activate [src].</span>")
+	apply_wibbly_filters(user)
+	if(do_after(user, 50, target = user) && user.cell.use(activationCost))
```
Comment (Sirryan2002) [inline code/game/objects/items/devices/chameleonproj.dm]:
```suggestion
	if(do_after(user, 5 SECONDS, target = user) && user.cell.use(activationCost))
```

Comment (GDNgit) [reply code/game/objects/items/devices/chameleonproj.dm]: I did that?
I think git is shitting the bed currently

Comment (S34NW) [reply code/game/objects/items/devices/chameleonproj.dm]: That was fixed there when I reviewed it, wtf how is it reverted

Comment (GDNgit) [reply code/game/objects/items/devices/chameleonproj.dm]: Git imploded but it's there now

**Review by Sirryan2002 — APPROVED** [review]
code looks good

**Pull request merged**